pkg/bindings/connection.go

```diff
@@ -60,3 +125,24 @@ func (c Connection) newRequest(httpMethod, endpoint string, httpBody io.Reader,
 	response, err := c.client.Do(req) // nolint
```

Comment:
We should re-initiate the server connection if dropped.  At least once, maybe three times...